test/integration/provisioning/destroy_all.rb

```diff
@@ -1,5 +1,3 @@
-require 'chef/provisioning'
-
```

Comment:
You shouldn't be removing this and the other ones. Looks like an errant script :wink: